SocketRocket.xcodeproj/project.pbxproj

```diff
@@ -658,6 +726,79 @@
 			};
 			name = Release;
 		};
+		3345DC8E1C52ACD70083CCB8 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
```

Comment:
Should be fine, same as all the other targets:
https://github.com/square/SocketRocket/blob/master/SocketRocket.xcodeproj/project.pbxproj